x-pack/plugins/enterprise_search/public/applications/shared/store/store.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getContext, kea, resetContext } from 'kea';
+
+resetContext({ createStore: true });
```

Comment:
to follow up on my above question - it looks like we could define some initial state here:

https://kea.js.org/docs/api/context

...but we'd need to turn `resetContext()` into a function so we can pass `http` into it 🤔